pkg/identity/numericidentity.go

```diff
@@ -583,9 +598,14 @@ func iterateReservedIdentityLabels(f func(_ NumericIdentity, _ labels.Labels)) {
 	}
 }
 
-// HasLocalScope returns true if the identity has a local scope
-func (id NumericIdentity) HasLocalScope() bool {
-	return (id & LocalIdentityFlag) != 0
+// HasLocalCIDRScope returns true if the identity is in the Local CIDR scope
+func (id NumericIdentity) HasLocalCIDRScope() bool {
+	return id.Scope() == IdentityScopeLocalCIDR
```

Comment:
nit: If we think this syntactic sugar is useful, may as well create corresponding helpers for `IdentityScopeLocalCIDR` as well.

Reply:
yeah, I added the equivalent function to make it more consistent.